Userland/Libraries/LibCore/Stream.h

```diff
@@ -1042,4 +1042,15 @@ class WrapInAKInputStream final : public InputStream {
     Core::Stream::Stream& m_stream;
 };
 
+// Note: This is only a temporary hack, to break up the task of moving away from AK::Stream into smaller parts.
```

Comment:
If you make this a FIXME, we can track it more easily.

Reply:
Tracking the fact that this class exists doesn't help much, the important part is the fact that this class is still used or will be used at some point down the line (to be more precise, potentially until all direct and indirect usages of `AK::Stream` are removed). More importantly, since there might be no usages of this class at a certain time, someone doing `FIXME`-roulette may just decide to remove it after finding no usages at that point in time.